product_docs/docs/eprs/7/eprs_rel_notes/eprs_rel_notes_7.8.0.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: "Replication Server 7.8.0 release notes"
+navTitle: Version 7.8.0
+---
+
+Released: 17 May 2024
+
+New features, enhancements, bug fixes, and other changes in Replication Server 7.8.0 include the following:
+
+| Type        | Description                                                                                                                                                                                                                                 | Addresses                  |
```

Comment:
The addition of the Addresses column is something new. Is this a new approach for identifying applicable support cases that is being adopted across the board for all products/tools?  Although I really like the format, if this is not something we're are doing for other products/tools, I would prefer the older style of noting the support case in brackets just to be consistent. If we are adopting this for all products/tools, I'm on board with this format.

Reply:
Hey @mlewandowski-edb ! 

Indeed, we are trying to streamline this format, which we are already applying to other products, like EPAS. 
I can create PR in the future to streamline the other release notes, so it looks consistent across versions.

Reply:
Sounds good.